pkg/models/model_plan.go

```diff
@@ -92,6 +92,33 @@ var ModelStatusHumanized = map[ModelStatus]string{
 	ModelStatusCanceled:              "Canceled",
 }
 
+var modelStatusChronologicalIndex = map[ModelStatus]int{
```

Comment:
Might wanna add one more comment here to this map as well -- just to clarify what it means, and if there's any specific gotchas (like why `Paused` and `Canceled` are at the end, just so people reading this for the first time / coming back to it can get a quick grip on what's being done!)